### PR TITLE
policy dao test added

### PR DESCRIPTION
**Pull request by oshhh**
Testing whether policy that satisfies criteria (active and of same category as pincode) is fetched. Checked that policies that don't satisfy the criteria are not fetched. 

TESTING:
All tests passing.

### CODE REVIEW

File: home_insurance_app/lib/data/policy_dao.dart [hunk 1]
```diff
@@ -2,9 +2,14 @@ import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:homeinsuranceapp/data/policy.dart';
 
 class PolicyDao {
```
Comment (akashagarwal) [inline home_insurance_app/lib/data/policy_dao.dart:4]: add comment

File: home_insurance_app/lib/data/policy_dao.dart [hunk 2]
```diff
@@ -2,9 +2,14 @@ import 'package:cloud_firestore/cloud_firestore.dart';
...
+  static var database;
```
Comment (dreamplay) [inline home_insurance_app/lib/data/policy_dao.dart:5]: why is this static

File: home_insurance_app/lib/data/policy_dao.dart [hunk 3]
```diff
@@ -2,9 +2,14 @@ import 'package:cloud_firestore/cloud_firestore.dart';
...
+  static setFirestoreInstance(var instance) {
+    database = instance;
+  }
+
   static Future<List<Policy>> getPolicies(int pincode) async {
```
Comment (dreamplay) [inline home_insurance_app/lib/data/policy_dao.dart:10]: getPolicies of PolicyDao shouldn't be static
static if for stateless methods.
database should be "injected" here as dependency

Comment (oshhh) [reply home_insurance_app/lib/data/policy_dao.dart:10]: Okay, I'll update that. Should I make it into a singleton?

**Pull request merged**